examples/counter/cli/src/main.rs

```diff
@@ -125,20 +146,14 @@ async fn http(uuid: Vec<u8>, method: Method, url: Url, tx: Sender<CoreMessage>)
 
     match status {
         200..=299 => match response.body_bytes().await {
```

Comment:
thinking about this, we probs shouldn't care about status here and just return it with the response regardless. Let the core decide what to do

Reply:
Gonna fix this on https://github.com/redbadger/crux/pull/50